src/luascript.cpp

```diff
@@ -8524,9 +8524,24 @@ int LuaScriptInterface::luaPlayerGetStorageValue(lua_State* L)
 	}
 
 	uint32_t key = getNumber<uint32_t>(L, 2);
-	int32_t value;
-	if (player->getStorageValue(key, value)) {
-		lua_pushnumber(L, value);
+	std::string strValue;
+	if (player->getStorageValue(key, strValue)) {
+		int32_t intValue;
+		try {
+			intValue = stoi(strValue);
+		} catch (std::invalid_argument& e) {
+			intValue = 0;
+		} catch (std::out_of_range& e) {
+			intValue = 0;
+		} catch (...) {
```

Comment:
If you are using `catch (...)`, why the two above `catch` statements doing the same?